parsers/ENTSOE.py

```diff
@@ -96,14 +113,10 @@ def query_consumption(domain, session, now=None):
     response = query_ENTSOE(session, params, now)
     if response.ok: return response.text
     else:
-        # Grab the error if possible
-        soup = BeautifulSoup(response.text, 'html.parser')
-        text = soup.find_all('text')
-        if len(text):
-            error_text = soup.find_all('text')[0].prettify()
-            if 'No matching data found' in error_text: return
-            raise Exception('Failed to get consumption. Reason: %s' % error_text)
-    
+        check_page = find_query_error(response)
+        if check_page:return
```

Comment:
Why not raise the error inside the `find_query_error` method (which we should rename to `check_response`)?
A bit like the `raise_for_status` in python requests

Reply:
Sure I was just following the original format but that's easy to implement.

Reply:
thanks!